src/minimum_dependencies/_core.py

```diff
@@ -72,11 +72,10 @@ def minimum_version(requirement: Requirement, fail: Fail = Fail.FALSE) -> Versio
     The minimum version available on PyPi for the given requirement.
     """
     if not requirement.specifier:
-        msg = f"No version specifier for {requirement.name} in install_requires."
+        msg = f'Could not parse a version specifier from "{requirement}"'
```

Comment:
```suggestion
        msg = f'Could not parse a version specifier from {requirement.name}'
```

Use the name, since we don't have the specifier in this case and the `extra` information is meaningless in this context. I also find the quotes distracting.